ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobStore.java

```diff
@@ -174,10 +174,10 @@ private boolean shouldUpload(MessageInfo messageInfo) {
     if (messageInfo.isDeleted()) {
       return false;
     }
-
-    // expiration time above threshold
+    // expiration time above threshold. Expired blobs are blocked by ReplicaThread.
```

Comment:
Update the comment to include "or is a ttl update"

Reply:
added.